docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -0,0 +1,54 @@
+Referencing Content Blocks in Library V2
+--------------------------------------------------
+
+Status
+=======
+Pending
+
+Context
+=======
+Currently, the courses are being stored in the Modulestore and we want the library content to be stored in the Blockstore.
+To achieve this we need a referencing mechanism to reference the libraries to the courses. This referencing helps in replacing
+v1 libraries (current content libraries stored on Modulestore) with v2 libraries (Blockstore backed) which are:
```

Comment:
I would suggest re-phrasing some of this:

> Currently, courses are stored in Modulestore (MongoDB), and libraries can be stored in either Modulestore (libraries v1) or blockstore (libraries v2). We are currently building out the functionality of libraries v2, and trying to migrate all usage of content libraries onto Blockstore.

Reply:
@saksham115 could you please make this edit? ^

Reply:
Please have a look if the edit makes sense :)